plugins/polio/api.py

```diff
@@ -241,6 +243,53 @@ def create_calendar_xlsx_sheet(self, request, **kwargs):
         response["Content-Disposition"] = "attachment; filename=%s" % filename + ".xlsx"
         return response
 
+    @action(methods=["GET"], detail=False, serializer_class=None)
+    def csv_campaign_scopes_export(self, request, **kwargs):
+        groups = Group.objects.filter(id__in=request.GET.get("group", None).split(","))
+        org_units_list = []
+        for group in groups:
+            org_units = OrgUnit.objects.filter(groups=group.id)
+            campaign_scope = CampaignScope.objects.get(group_id=group.id)
+            campaign = campaign_scope.campaign
+            rounds = Round.objects.filter(campaign_id=campaign.id)
+            for org_unit in org_units:
+                item = {}
+                item["id"] = org_unit.id
+                item["org_unit_name"] = org_unit.name
+                item["org_unit_parent_name"] = org_unit.parent.name
+                item["org_unit_parent_of_parent_name"] = org_unit.parent.parent.name
+                item["obr_name"] = campaign.obr_name
+                item["round_number"] = [round.number for round in rounds]
+                org_units_list.append(item)
+
+        filename = "%s--%s-%s" % ("campaigns", "org_units", strftime("%Y-%m-%d-%H-%M", gmtime()))
+        columns = [
+            {"title": "ID", "width": 10},
+            {"title": "Admin 2", "width": 25},
+            {"title": "Admin 1", "width": 25},
+            {"title": "Admin 0", "width": 25},
```

Comment:
I don't like this header much, but from what I see in the ticket, that's the request so 🤷‍♂️